Data.json

```
@@ -14,9 +14,14 @@
     "username": "vikasganiga05",
     "avatar": "https://github.com/vikasganiga05.png"
   },
-    {
+  {
     "name": "Shourjjo Majumder",
     "username": "shourgamer2",
     "avatar": "https://github.com/shourgamer2.png"
+  },
+  {
+    "name": "Ivan Eka Putra",
```

```suggestion
    "name": "Ivan Eka Putra",
```

Hi @ivanetra 

Thanks for your contribution, please keep the journey as a separate PR and fixing #50 as a separate PR.

Also, you haven't created a journey, you have only added the .json, please create a journey in the journeys directory.

the json helps the website link your journey properly over there, but is useless if it has nothing to link to :)

Thanks!

---

I am deleting the data.json code which you have added please create a journey and add this json code again in a new PR, sorry for the inconvenience. 

Thanks

---

Thank you @KendallDoesCoding !